src/plugins/intel_gpu/src/graph/program.cpp

```diff
@@ -1445,7 +1445,9 @@ void program::set_layout_optimizer_attributes(layout_optimizer& lo) {
             prim.type() != cldnn::prior_box::type_id() &&
             prim.type() != cldnn::resample::type_id() &&
             prim.type() != cldnn::eye::type_id() &&
-            prim.type() != cldnn::generate_proposals::type_id()) {
+            prim.type() != cldnn::generate_proposals::type_id() &&
+            prim.type() != cldnn::bucketize::type_id() &&
```

Comment:
Seems that it is not rebased. Please rebase.

Reply:
done